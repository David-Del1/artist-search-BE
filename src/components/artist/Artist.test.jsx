import React from 'react';
import { render } from '@testing-library/react';

describe('album component', () => {
  it('makes a album', () => {
    const { asFragment } = render(
      <section>
        <p>Sara</p>
        <p>5</p>
      </section>
    );

    expect(asFragment()).toMatchSnapshot();
  });
});
