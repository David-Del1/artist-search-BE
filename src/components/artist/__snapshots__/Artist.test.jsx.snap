// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`album component makes a album 1`] = `
<DocumentFragment>
  <section>
    <p>
      Sara
    </p>
    <p>
      5
    </p>
  </section>
</DocumentFragment>
`;
